src/ngx_rewrite_driver_factory.cc

```diff
@@ -151,6 +151,10 @@ RewriteOptions* NgxRewriteDriverFactory::NewRewriteOptions() {
   return options;
 }
 
+RewriteOptions* NgxRewriteDriverFactory::NewRewriteOptionsForQuery() {
+  return new NgxRewriteOptions(thread_system());
```

Comment:
> the only place in Apache where we establish a default of kCoreFilters

There's also `RewriteDriverFactory::InitializeDefaultOptions`, which applies to both Apache and Nginx, and I see it running in Nginx.  The problem is that it does `SetDefaultRewriteLevel(kCoreFilters)` but for some reason I still don't see Nginx needs `SetRewriteLevel`.